src/api/api.ts

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios';
+
+export const client = axios.create({
+  baseURL: 'https://mate.academy/students-api',
+})
+
+//#region User
+export const getUserByEmail = (email: string) => {
+  return client.get('/users', {
+    params: {
+      email,
+    }
+  })
+}
+
+export const createUser = (email: string, name: string) => {
+  return client.post('/users', {
+    email,
+    name,
+  })
+}
+//#endregion
+
+//#region Posts
+export const getPostsById = (userId: number) => {
+  return client.get('/posts', {
+    params: {
+      userId,
+    }
+  })
+}
```

Comment:
The function name getPostsById suggests that it should return a single post by its id, but it seems to return multiple posts by user id. Consider renaming it to getPostsByUserId for better clarity.

Reply:
done